recipes/gmap/2016.09.23/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: gmap
+  version: "2016.09.23"
```

Comment:
Version 2016-09-23 should work for most people. However, on some scarce occasions (when a fasta looks in a very particular way) it will crash. It is not possible to go around this using any of the available parameters.
2017-02-15 takes care of the above.